### ISSUE TITLE
Increase Default Toast Duration

### ISSUE THREAD

**Issue by earth2travis** [HausDAO/daohaus-monorepo#604]
Toast messages are not staying up long enough to read them. They are currently set at 2 seconds and should be increased to follow accessibility best practices

https://sheribyrnehaber.medium.com/designing-toast-messages-for-accessibility-fb610ac364be

**Comment by earth2travis** [comment]
Increased to 6 seconds
